<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialCTScanExperimentsClass</class>
 <widget class="QMainWindow" name="SerialCTScanExperimentsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialCTScanExperiments</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxSerial">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Serial COM port</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="SerialToArduino" name="serialWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Light</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonTimer">
         <property name="text">
          <string>Read timer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton3">
         <property name="text">
          <string>&lt;LED3--&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton1">
         <property name="text">
          <string>&lt;LED1--&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton2">
         <property name="text">
          <string>&lt;LED2--&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QSpinBox" name="spinBoxTimer">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Logging</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Sensors</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QProgressBar" name="progressBar3">
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="progressBar1">
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="progressBar2">
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="progressBar4">
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSpinBox" name="spinBox1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spinBox2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBox3">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="spinBox4">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1334</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SerialToArduino</class>
   <extends>QWidget</extends>
   <header>serialtoarduino.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SerialCTScanExperiments.qrc"/>
 </resources>
 <connections/>
</ui>
